sprocs/instance_questions_update_in_grading.sql

```diff
@@ -1,25 +0,0 @@
-CREATE FUNCTION
-    instance_questions_update_in_grading (
```

Comment:
Can you explain why this sproc is no longer needed?

Reply:
It was never used to begin with. Also, the `_in_grading` fields were updated, but never used in any API, UI or meaningful result. There's no clear documentation as to what these fields were supposed to represent (I believe it's the number of points that can still be achievable, but there are other better ways to compute that).